.github/workflows/build_test_deploy.yml

```
@@ -263,44 +263,58 @@ jobs:
         run: docker compose --file .github/docker-compose.yml down --volumes --remove-orphans
 
       - uses: docker/login-action@v3
+        if: ${{ github.actor != 'dependabot[bot]' &&
+          (github.event_name != 'pull_request' || github.event.pull_request.head.repo.full_name == github.repository) }}
         with:
           username: ${{ secrets.DOCKER_USER }}
           password: ${{ secrets.DOCKER_PASS }}
 
       - name: Push webknossos image
-        if: ${{ ! startsWith(github.ref, 'refs/tags/') }}
+        if: ${{ ! startsWith(github.ref, 'refs/tags/')
```

actionlint (1.7.7)

parser did not reach end of input after parsing the expression. 29 remaining token(s) in the input: "IDENT", ".", "IDENT", "!=", "STRING", "&&", "(", "IDENT", ".", "IDENT", "!=", "STRING", "||", "IDENT", ".", "IDENT", ".", "IDENT", ".", "IDENT", ".", "IDENT", ".", "IDENT", "==", "IDENT", ".", "IDENT", ")"

(expression)